src/__test__/components/data-exploration/differential-expression-tool/DiffExprResults.test.jsx

```diff
@@ -179,8 +179,8 @@ describe('DiffExprResults', () => {
       </Provider>,
     );
 
-    const table = component.find('Table Table');
-    const spin = component.find('Table').find(Loader);
```

Comment:
The new `Table` element in the updated `Antd` version (4.17.2) does not wrap its contents inside a `Table` element anymore, causing these tests to fail.